pkg/types/types.go

```diff
@@ -517,6 +520,54 @@ type Connection struct {
 	Verify   string `yaml:"verify,omitempty"`
 }
 
+type AuditConfig struct {
+	Disabled         bool             `yaml:"disabled,omitempty"`
+	PolicyFile       string           `yaml:"auditPolicyFile,omitempty"`
+	APIServerOptions APIServerOptions `yaml:"kubeApiServerOptions,omitempty"`
```

Comment:
I don't think we need this -we can use sane defaults, and then use the apiServer.extraArgs to override: 

e.g.

```yaml
kubernetes:
    auditing:
       policyFile: ./policy.yaml
    apiServerExtraArgs:
       "audit-log-maxsize": 1GB
```